build_byod_snapshot.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+
+set -uo pipefail
+
+# planned improvement will be commented in the following task
+# https://anyscale1.atlassian.net/browse/AE-529
+
```

Comment:
can you also put dependencies in of this shell script in the comment?
e.g., docker, 100G of disk space etc...